ldk/go/keyboardService.go

```diff
@@ -20,11 +20,12 @@ func (h Hotkey) Match(v Hotkey) bool {
 		return false
 	}
 	// You can't do a direct bitmask as Alt means Altleft or AltRight.  So checking all combinations is the only safe way to actually compare
```

Comment:
Is this true only for Alt?  Or is this true for every left/right modifier?

Reply:
I'm not 100% sure what the the bitmask shifting is doing, so I don't know if I can answer that